akka-stream/src/main/scala/akka/stream/javadsl/japi/WithVariance.scala

```diff
@@ -6,6 +6,15 @@ package akka.stream.javadsl.japi
 // TODO Same SAM-classes as in akka.japi, but with variance annotations
 // TODO Remove these in favour of using akka.japi with added variance
 
+object Function {
+  final val Identity = new Function[Any, Any] {
```

Comment:
Hm, while in theory it should be fine I can't seem to get it to compile (`param: Any): Nothing = param` => nope)